examples/bounce.js

```diff
@@ -37,9 +37,9 @@ function rain (drops) {
    }
 }
 
-function demo (n) {
+function demo(n) {
    wrap(false);
    hideTurtle();
-   var drops = init_drops(n);
-   animate(function() { rain(drops) }, 100);
+   const drops = init_drops(n);
+   animate(() => { rain(drops) }, 100);
```

Comment:
B/c it's 1 statement we don't need {} for the fat arrow lambda:
`animate(() => rain(drops), 100);`

Reply:
I like the idea but I don't "approve" it, because IDK if @bjpop would prefer that. The arrow fn is supposed to "`return void`", so the brackets make it "more explicit"

Reply:
IMO the unnecessary brackets merely makes the lambda expression larger than it has to be.
99.9% of callbacks' returning value (or lack of it) is ignored anyways.
Therefore we shouldn't worry about making `void` returning datatype explicit for callbacks.

Reply:
Good point 🤔. I'll commit the change, and if bjpop doesn't want it, I'll revert it